mosaic/datasets.py

```diff
@@ -0,0 +1,91 @@
+import pathlib
+
+import pooch
+import xarray as xr
+from xarray import Dataset
+
+from mosaic.version import __version__
+
+# this dictionary uses the short handed mesh names for convience,
+# but can not be parsed by pooch
+registry = {
+    "QU.960km": {
+        "lcrc_path": "mpas_standalonedata/mpas-ocean/mesh_database/mesh.QU.960km.151026.nc",
+        "sha256_hash": "sha256:524d2d5a93851395a3bdfafb30a5acb2a3dbbecd0db07cd29e5e3f87da6eb82f"
+    },
+
+    "QU.240km": {
+        "lcrc_path": "inputdata/ocn/mpas-o/oQU240/ocean.QU.240km.151209.nc",
+        "sha256_hash": "sha256:a3758f88ceff3d91e86dba7922f6dd7d5672157b4793ef78214624ab8b2724ae"
+    },
+
+    "mpaso.EC30to60E2r3": {
+        "lcrc_path": "inputdata/ocn/mpas-o/EC30to60E2r3/mpaso.EC30to60E2r3.230313.nc",
+        "sha256_hash": "sha256:55e7cc33c890f7b9f1188bcc07fd8218b57cb1cd5ba32ee66fe3162a36995a7c"
+    },
```

Comment:
I would use the Icos30E3r5 mesh instead:
https://web.lcrc.anl.gov/public/e3sm/inputdata/share/meshes/mpas/ocean/IcoswISC30E3r5.20231120.nc
This is just the base horizontal mesh, not a full ocean initial condition, so it's ~1.2 GB rather than 3.0 GB.

I suggest this because we are no longer using `EC30to60E2r3` in E3SM v3.